packages/app-runtime/test/runtime/Onboarding.test.ts

```diff
@@ -82,4 +84,24 @@ describe("Onboarding", function () {
         });
         expect(recoveryKitResponse2).toBeSuccessful();
     });
+
+    test("should onboard with an anonomously created empty token", async () => {
+        const emptyToken = await onboardingRuntime.anonymousServices.tokens.createEmptyToken();
+
+        const device = await services.transportServices.devices.createDevice({});
+        const onboardingInfo = await services.transportServices.devices.getDeviceOnboardingInfo({ id: device.value.id });
+
+        const tokenContent = TokenContentDeviceSharedSecret.from({ sharedSecret: DeviceMapper.toDeviceSharedSecret(onboardingInfo.value) });
+        const result = await services.transportServices.tokens.updateTokenContent({ reference: emptyToken.value.reference.truncated, content: tokenContent.toJSON() });
+        expect(result).toBeSuccessful();
```

Comment:
I would probably prefer a single method doing all of this (=> fetching the token, creating the content, updating the token), so that the user of the app runtime doesn't know how to do it.